components/org.wso2.carbon.identity.application.authenticator.fido/src/main/java/org/wso2/carbon/identity/application/authenticator/fido/connector/FIDOAuthenticatorConfigImpl.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.application.authenticator.fido.connector;
+
+import org.apache.commons.lang.StringUtils;
+import org.wso2.carbon.identity.application.authenticator.fido.util.FIDOAuthenticatorConstants;
+import org.wso2.carbon.identity.core.util.IdentityUtil;
+import org.wso2.carbon.identity.governance.IdentityGovernanceException;
+import org.wso2.carbon.identity.governance.common.IdentityConnectorConfig;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+/**
+ * This class contains the authenticator config implementation.
+ */
+public class FIDOAuthenticatorConfigImpl implements IdentityConnectorConfig {
+    @Override
+    public String getName() {
+        return FIDOAuthenticatorConstants.AUTHENTICATOR_NAME;
+    }
+
+    @Override
+    public String getFriendlyName() {
+        return FIDOAuthenticatorConstants.AUTHENTICATOR_FRIENDLY_NAME;
+    }
+
+    @Override
+    public String getCategory() {
+        return "Multi Factor Authenticators";
+    }
+
+    @Override
+    public String getSubCategory() {
+        return "DEFAULT";
+    }
+
+    @Override
+    public int getOrder() {
+        return 0;
+    }
+
+    @Override
+    public Map<String, String> getPropertyNameMapping() {
+        Map<String, String> nameMapping = new HashMap<>();
```

Comment:
Add a new line after the method signature. Please check all other places as well.

Reply:
Resolbed via [bb710a4](https://github.com/wso2-extensions/identity-local-auth-fido/pull/110/commits/bb710a424ea12ba95dcb83f8274a3ad825f99f69)